code/modules/cargo/packs.dm

```diff
@@ -403,6 +383,29 @@
 					/obj/item/storage/box/handcuffs)
 	crate_name = "security supply crate"
 
+/datum/supply_pack/security/disabler
+	name = "Disabler Crate"
+	desc = "Three stamina-draining disabler weapons. Requires Security access to open."
+	cost = 1500
```

Comment:
Sugerowałbym zwiększenie kosztu disablerów jak wprowadzamy balistyki, przez ich niską cene lepiej jest kupić to niż pistolety ochroniarskie. ED 209 i turrety są budowane raz na nigdy więc to nie ucierpi zbytnio.

Reply:
Fair point